pkg/log/log.go

```diff
@@ -0,0 +1,49 @@
+package log
+
+import "github.com/sirupsen/logrus"
+
+var logger = (*logrus.Logger)(nil)
```

Comment:
Will this also perform the fmt. commands (fmt.printf, fmt.println...)?
If not then I think each function should also contain that so we dont lose debugging functionality.

Reply:
Yes it replaces the current fmt.Print and log.Info